### PR TITLE
RUN-12872-workloads-new-page

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/admin/workloads/workload-overview-admin.md [hunk 1]
```diff
@@ -2,25 +2,54 @@
...
 | Inference      | `InferenceWorkload` | Submit an inference workload |
 
+## Workloads View
+
+The *Workloads* view provides a more advanced UI than the previous *Jobs* UI.
+
+To enable the *Workloads* view, go to the *Jobs* table and toggle the switch from `Jobs` to `Workloads`. To return, switch the toggle from `Workloads` to `Jobs`.
+
+!!! Note
+    To submit a workload, use either the *Jobs* view or the CLI.
```
Comment (lavianalon) [inline docs/admin/workloads/workload-overview-admin.md:28]: There will be an option to submit a Workspace or a Trainings workload from the new view.

Comment (jasonnovichRunAI) [reply docs/admin/workloads/workload-overview-admin.md:28]: I am not seeing it in 2.15 - when this changes I will add it.

```suggestion
    To submit a workload, use either the *Jobs* view or the CLI.
    
    <!-- remove this when the feature is added -->
```

**Comment by jasonnovichRunAI** [comment]
@omerbenedictrunai for your review. If ok please approve the PR for merging.

**Review by omerbenedictrunai — COMMENTED** [review]
Line 39 - new workloads does not require selection of a line.

**Comment by jasonnovichRunAI** [comment]
> Line 39 - new workloads does not require selection of a line.

Yes, I will fix this. I also need to add the procedure for adding a new workload. I now have a env which can do that. I will tag you for re-review once that is complete.

**Review by lavianalon — APPROVED** [review]

**Pull request merged**